vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -278,6 +285,10 @@ class LEDGER_EXPORT Ledger {
       const uint64_t to_timestamp_seconds,
       ledger::ConfirmationsHistoryCallback callback) = 0;
   virtual void GetRewardsInternalsInfo(ledger::RewardsInternalsInfo* info) = 0;
+
+  virtual void RemoveData(int32_t remove_mask,
+      ledger::OnRewardsRemovedDataCallback callback) = 0;
```

Comment:
nit: the callback doesn't need Rewards either. Typically we've been following a convention like `OnDataRemovedCallback`, but I'm also fine with `OnRemoveDataCallback`. Also ledger namespace isn't needed her

Reply:
Fixed